### PR TITLE
Add WNP 133 for VPN (Fixes #15488)

### LINKED ISSUE #15488: WNP (Version 4)
## Description

VPN has expanded to more regions. We will be advertising to the regions that we are expanding to. These regions are not a part of the target audience for the other WNP and should be safe to geo target.

### Audience
**Countries:** India, Indonesia, Brazil, Mexico, Türkiye, South Africa, Vietnam, Saudi Arabia, Thailand, Uganda, Senegal, Australia, Bangladesh, Ukraine, Egypt, Greece, Nigeria, Morocco, Colombia, Kenya, Chile, South Korea, Taiwan, and Norway.

**Languages available:** English, Portuguese, Chinese, Spanish , French, Ukrainian, Greek, Vietnamese, Turkish, Indonesian, and South Korean.

**Languages Codes:** EN-*, zh-TW, vi-VN, uk-UA, tr-TR, pt-BR, ko-KR, id-ID, fr-FR, es-LA, el-GR

**Breakdown:** 100%

Design: [Link](https://www.figma.com/design/CZvzHgU9NQMQUm34lomIl5/Firefox-WNP?node-id=9285-17328&node-type=canvas&t=a7h7YzxB6P7ubZPv-0) (Dark mode optional, not necessary)
Copy: [Link](https://drive.google.com/drive/folders/1uCQHsd7noa897JIKusfbV2KFMkuxlG5o)

#### Notes: 
- Needs to be Geo-targeted. We need to confirm how this works with Nimbus
- This is based on a previous WNP design. Possible to reuse older code.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP 133 for VPN wave VII launch countries / languages.

## Issue / Bugzilla link

#15488

## Testing

Please test both light and dark mode:

- http://localhost:8000/en-US/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/el/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/es-ES/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/fr/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/id/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/ko/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/pt-BR/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/tr/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/uk/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/vi/firefox/133.0/whatsnew/?geo=AU
- http://localhost:8000/zh-TW/firefox/133.0/whatsnew/?geo=AU

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        {
+          "media": "(prefers-color-scheme: dark)",
+          "srcset": {
+            "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-header-dark.svg": "default",
+          }
+        }
+      ],
+      optional_attributes={
+        "class": "wnp-main-image",
+        "alt": "Firefox",
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html:127]:
```suggestion
        "alt": "Mozilla VPN",
```

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  {% set picto_1_heading = 'Jangan korbankan kecepatan'|safe %}
+  {% set picto_1_desc = 'Streaming, unduh, dan mainkan game tanpa hambatan, berkat server yang didukung <strong>WireGuard®</strong>.'|safe %}
+  {% set picto_2_heading = 'Rasakan ketenangan'|safe %}
+  {% set picto_2_desc = 'Jelajahi internet tanpa meninggalkan jejak — kami tidak pernah mencatat atau melacak data jaringan Anda.'|safe %}
+{% elif LANG == 'ko' %}
+  {% set main_title = '클릭 한 번으로 경험하는 Mozilla VPN의 개인정보 보호기능'|safe %}
+  {% set main_tagline = '<strong>Mozilla VPN</strong>(가상 사설 네트워크)이 컴퓨터, 모바일 기기, 태블릿에서 인터넷 연결 전체를 보호합니다.'|safe %}
+  {% set main_cta = '지금 Mozilla VPN 이용하기'|safe %}
+  {% set picto_1_heading = '속도, 양보하지 마세요'|safe %}
+  {% set picto_1_desc = 'WireGuard® 서버로 스트리밍, 다운로드, 게이밍을 속도 저하 없이 즐기세요.'|safe %}
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html:53]: **question**: is the bolding deliberately omitted here?

```suggestion
  {% set picto_1_desc = '<strong>WireGuard®</strong> 서버로 스트리밍, 다운로드, 게이밍을 속도 저하 없이 즐기세요.'|safe %}
```

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  {% set picto_1_heading = 'Не втрачайте швидкість'|safe %}
+  {% set picto_1_desc = 'Стримінг, завантаження та ігри без затримок завдяки нашим серверам із <strong>WireGuard®</strong>.'|safe %}
+  {% set picto_2_heading = 'На варті вашого спокою'|safe %}
+  {% set picto_2_desc = 'Користуйтесь інтернетом, на залишаючи цифрових слідів, адже ми ніколи не зберігаємо і не відстежуємо ваші мережеві дані.'|safe %}
+{% elif LANG == 'vi' %}
+  {% set main_title = 'Dễ dàng bảo vệ quyền riêng tư  với Mozilla VPN'|safe %}
+  {% set main_tagline = '<strong>Mozilla VPN</strong> (Mạng Ảo Riêng Tư) bảo vệ toàn bộ kết nối mạng trên máy tính, điện thoại và máy tính bảng.'|safe %}
+  {% set main_cta = 'Dùng Mozilla VPN Ngay'|safe %}
+  {% set picto_1_heading = 'Tốc độ không suy giảm'|safe %}
+  {% set picto_1_desc = 'Phát, tải xuống và chơi game không gián đoạn nhờ hệ thống máy chủ WireGuard® mạnh mẽ của chúng tôi.'|safe %}
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html:85]: **question**: is the bolding deliberately omitted here

```suggestion
  {% set picto_1_desc = 'Phát, tải xuống và chơi game không gián đoạn nhờ hệ thống máy chủ <strong>WireGuard®</strong> mạnh mẽ của chúng tôi.'|safe %}
```

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+        ),
+        body=True,
+      ) %}
+        <p>{{ picto_2_desc }}</p>
+      {% endcall %}
+    </ul>
+  </div>
+</section>
+
+<section class="wnp-footer">
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html:206]: **nit, non-blocking**: might be worth including inline padding to match the sections above and below `--h-grid-xs` or a `mzp` container?

Screenshot 2024-11-18 at 11 18 53 AM

**Review by maureenlholland — APPROVED** [review]
💫 so many languages and countries! Nothing blocking here. Light/Dark mode looks great

- 2 very minor typos
- 2 not-sure-if-deliberate omissions of strong tag

**Comment by alexgibson** [comment]
Thanks for the eagle eyes @maureenlholland, nice suggestions 👍

**Pull request merged**